modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -30,58 +30,58 @@
     <test name="is-tests-default-configuration" preserve-order="true" parallel="false" group-by-instances="true">
         <classes>
             <class name="org.wso2.identity.integration.test.oauth2.OAuth2DeviceFlowTestCase"/>
-
```

Comment:
Why are test cases commented out?

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/38e6cdd0e3f7357405e8fe1891714dcfcb17624b